src/main/java/org/commcare/session/RemoteQuerySessionManager.java

```diff
@@ -226,6 +227,20 @@ public void refreshItemSetChoices() {
         }
     }
 
+
+    public void validateUserAnswers() {
```

Comment:
`errors` should be cleared at the start of this method

Reply:
a test for this would be good to ensure that errors are resolved after user input is corrected